_readme_templates.gotmpl

```diff
@@ -0,0 +1,18 @@
+{{ template "chart.header" . }}
+{{ template "chart.deprecationWarning" . }}
+
+{{ template "chart.badgesSection" . }}
+
+{{ template "chart.description" . }}
+
+This helm chart is maintain and released by the fluxcd-community on a best effort basis.
```

Comment:
@stefanprodan We can add some kind of community disclaimer here and it will be rendered to all chart Readme's. Feel free to adjust it in a next PR if this is not enough.